baseapp/baseapp.go

```diff
@@ -414,6 +413,7 @@ func (app *BaseApp) setState(mode runTxMode, header cmtproto.Header) {
 		ms:  ms,
 		ctx: sdk.NewContext(ms, header, false, app.logger),
 	}
+	baseState.ctx = baseState.ctx.WithStreamingManager(app.streamingManager)
```

Comment:
Why not just move the `WithStreamingManager(app.streamingManager)` to line 414

Reply:
moved to line 414.